systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -138,10 +147,54 @@ void testTieredStorageWithAivenPlugin() {
         resourceManager.createResourceWithWait(clients.producerStrimzi());
 
         SetupMinio.waitForDataInMinio(suiteStorage.getNamespaceName(), BUCKET_NAME);
+
+        // Create admin-client to check offsets
+        resourceManager.createResourceWithWait(
+            AdminClientTemplates.plainAdminClient(
+                testStorage.getNamespaceName(),
+                testStorage.getAdminName(),
+                KafkaResources.plainBootstrapAddress(testStorage.getClusterName())
+            )
+            .editSpec()
+                .editOrNewTemplate()
+                    .editSpec()
+                        .editFirstContainer()
+                            // TODO - remove this when new version of clients will be available
+                            .withImage("quay.io/strimzi-test-clients/test-clients:latest-kafka-3.7.1")
```

Comment:
Should it be just for 3.7.1? Or should it be handled by the `ST_KAFKA_VERSION` env variable (for now until the new test-clients version is released)?

Reply:
Added ST_KAFKA_VERSION, guess it should work on all of them, however, -5L option will work only on 3.7+ I think